### PR TITLE
Api definition for nakadi jvm

### PR DESCRIPTION
**Pull request by tmuehl**
@valgog @v-stepanov @jkliff @CyberDem0n Please review and comment

Unfortunately I had to remove the securityDefinition because the tokelUrl and authorizationUrl are mendatory.
Some changes might raise some controversy, please see this as my suggestion for now. If you don't agree, raise your voice.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/39" target="_blank">ReviewNinja

File: nakadi/swagger.yaml [hunk 1]
```diff
@@ -1,45 +1,33 @@
...
-    * High level API should support automatic redistribution of the event streams
-    during client topology changes.
-
-  version: '0.3'
+  version: '0.4'
   contact:
-    name: Team Aruha
-    email: team-aruha@zalando.de
+    name: Team Aruha @ Zalando
+    email: nakadi-users@zalando.de
```
Comment (v-stepanov) [inline nakadi/swagger.yaml]: Why you changed it? I actually thought that nakadi-users@zalando.de is mailing list our stakeholders will be subscribed to.

File: nakadi/swagger.yaml [hunk 2]
```diff
@@ -1,45 +1,33 @@
...
 schemes:
   - https
 consumes:
   - application/json
 produces:
   - application/json
-securityDefinitions:
```
Comment (v-stepanov) [inline nakadi/swagger.yaml]: Why you removed securityDefinitions? We actually do support that.

File: nakadi/swagger.yaml [hunk 3]
```diff
@@ -1,45 +1,33 @@
...
-      uid: Unique identifier of the user accessing the service.
+
 paths:
   /metrics:
     get:
       tags:
         - monitoring
       summary: Get monitoring metrics
-      operationId: nakadi.hack.get_metrics
+      operationId: nakadi.metrics.get
```
Comment (v-stepanov) [inline nakadi/swagger.yaml]: Didn't you want to throw that away?

**Pull request closed without merging**